docs/source/tutorials_source/package/tutorial_pretrain_detectron2.py

```diff
@@ -261,7 +261,7 @@ def forward(self, x):
 
 # %%
 #
-# The :py:class:`lightly.data.collate.SimCLRCollateFunction` applies an ImageNet
+# The :py:class:`lightly.transforms.simclr.SimCLRTransform` applies an ImageNet
```

Comment:
```suggestion
# The :py:class:`~lightly.transforms.simclr.SimCLRTransform` applies an ImageNet
```

This will show only `SimCLRTransform` instead of `lightly.transforms.simclr.SimCLRTransform` in the text.